### ISSUE TITLE
Extra Credit: Concurrency Game

### ISSUE THREAD

**Issue by mikeizbicki** [mikeizbicki/cmc-csci143#482]
<https://deadlockempire.github.io/> is a game about managing concurrency in C#. The language is obviously different than the sql/python combination we covered in class, but otherwise the principles are the same.

Therefore, I will award extra credit for completing the game. If you complete everything up to (and including) the "Locks" section, you will get +1 point ec. If you finish the entire game, you will get an additional +1 point ec.

These extra sections cover alternatives to locks that modern programming languages and operating systems support.  Unfortunately, they are not implemented in standard SQL implementations.  I would be excited to supervise a student thesis/research project on implementing some of these other methods as a library in Postgres.

**Comment by abraryaser02** [comment]
Hi Mike! Fortunately was able to complete the game up to the locks section because of some C I learned in my OS class. Do I just show you the physical proof during OH? Thanks.

**Comment by mikeizbicki** [comment]
@abraryaser02 Yes.  It has to be shown to me in person.  Office hours/after class/during the final exam timeslot are all fine times.
